src/components/Carousel/Carousel.css

```diff
@@ -69,4 +70,62 @@
 .carousel-item__spec-type {
     font-weight: 400;
     color: #525252;
+}
```

Comment:
We need another media query to handle resolutions greater than 1280:

![image](https://user-images.githubusercontent.com/37159177/112775941-4796bb00-900c-11eb-9e73-e9ab658fc990.png)